tests/mock_libraries/imu/mock_i2c.hpp

```diff
@@ -11,3 +14,5 @@ class MOCKI2C : public I2CInterface {
   MOCK_METHOD(types::DriverStatus, Write, (std::uint8_t address, const std::vector<std::uint8_t>& data, std::uint32_t timeout), (noexcept));
```

Comment:
Why is Mock class all CAPS?

Reply:
Good question. Fixed here: https://github.com/Squiddrone/Squiddrone/pull/269/commits/5d0fcb5c8f5199d935733c0de6c800b7b0092e2f